src/winforms/toga_winforms/window.py

```diff
@@ -44,6 +44,8 @@ def create(self):
         self.native.Resize += self.winforms_resize
         self.toolbar_native = None
         self.toolbar_items = None
+        self.native.StartPosition = 0   # FormStartPosition.Manual
```

Comment:
What exactly does this statement do? Setting Location makes sense; but it isn't clear why "StartPosition = 0" is needed (and the comment at the end of the line doesn't make much sense either). A clarifying comment about exactly what is needed would be helpful here, if only to explain to future explorers what is going on.

Reply:
In Winforms, FormStartPosition is an enum which defines how the new window should be positioned: https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.formstartposition?view=netcore-3.1

Other values would be 4 (center on parent) or 1 (center on screen). Setting it to 0 allows you to set the position by x/y coordinates

Reply:
Ah - if FormStartPosition is an enumeration, then we should be using that symbol directly from Winforms. You should find `WinForms.FormStartPosition` is available as a symbol directly.

Reply:
OK, used WinForms.FormStartPosition.Manual instead of 0 and added an explaining comment